cpp/include/cudf/io/parquet.hpp

```diff
@@ -567,9 +585,25 @@ class parquet_writer_options {
   {
     CUDF_EXPECTS(
       size_rows >= 5000,
-      "The maximum row group size cannot be smaller than the page size, which is 5000 rows.");
+      "The maximum row group size cannot be smaller than the fragment size, which is 5000 rows.");
     _row_group_size_rows = size_rows;
   }
+
+  /**
+   * @brief Sets the maximum uncompressed page size, in bytes.
+   */
+  void set_max_page_size_bytes(size_t size_bytes) { _max_page_size_bytes = size_bytes; }
+
+  /**
+   * @brief Sets the maximum page size, in rows.
+   */
+  void set_max_page_size_rows(size_type size_rows)
+  {
+    CUDF_EXPECTS(
+      size_rows >= 5000,
```

Comment:
This should be lower, IIUC 5K is the original size. ORC uses 512 here, maybe Parquet can do the same.

Reply:
I kept 5000 since that's the size of a fragment.  I don't think setting lower than 5000 will have any effect.